pyproject.toml

```diff
@@ -17,7 +17,7 @@ include = "packages"
 [tool.poetry.dependencies]
 python = "<3.12,>=3.8"
 open-autonomy = "==0.14.14.post1"
-requests = "==2.28.1"
+requests = "<2.31.2,>=2.28.1"
```

Comment:
Did you face some issues with the version?

Reply:
No, I didn't change this myself. I think it got changed when running a command. I can revert this.